library/std/src/panicking.rs

```diff
@@ -518,12 +523,12 @@ pub fn begin_panic_handler(info: &PanicInfo<'_>) -> ! {
 #[rustc_do_not_const_check] // hooked by const-eval
 pub const fn begin_panic<M: Any + Send>(msg: M) -> ! {
     if cfg!(feature = "panic_immediate_abort") {
-        intrinsics::abort()
+        crate::sys::abort_internal();
```

Comment:
`abort_internal()` is more heavy weight than `intrinsics::abort()` as it often results in a call: https://github.com/rust-lang/rust/blob/9e8356c6adf119f983651d533d2b307544086cf9/library/std/src/sys/unix/mod.rs#L259

Reply:
I personally feel that 4 extra bytes (x86 call is 5 bytes, int3 is 1 byte) is a good price to pay to get a SIGABRT instead of a very confusing SIGILL.

However it's not strictly necessary as part of this PR, so I can leave it out.